src/state/user/reducer.ts

```diff
@@ -9,6 +9,8 @@ import { SerializedPair, SerializedToken } from './types'
 const currentTimestamp = () => new Date().getTime()
 
 export interface UserState {
+  buyFiatFlowCompleted: boolean | undefined
```

Comment:
Simplify (because `undefined` is your stand-in for `false`, and a user can never set this to `false`), and propagate through the PR:

```suggestion
  buyFiatFlowCompleted?: true
```